src/locale/locales/nl/messages.po

```diff
@@ -0,0 +1,7903 @@
+msgstr "Blader door andere feeds"
+
+#: src/view/com/auth/SplashScreen.web.tsx:167
+msgid "Business"
+msgstr "Zakelijk"
+
+#: src/screens/Signup/StepInfo/Policies.tsx:81
+msgid "By creating an account you agree to the <0>Privacy Policy</0>."
+msgstr "Door een account aan te maken gaat u akkoord met het <0>Privacybeleid</0>."
+
+#: src/screens/Signup/StepInfo/Policies.tsx:68
+msgid "By creating an account you agree to the <0>Terms of Service</0>."
+msgstr "Door een account aan te maken gaat u akkoord met de <0>Servicevoorwaarden</0>."
+
+#: src/screens/Signup/StepInfo/Policies.tsx:48
+msgid "By creating an account you agree to the <0>Terms of Service</0> and <1>Privacy Policy</1>."
+msgstr "Door een account aan te maken stemt u in met de <0>Servicevoorwaarden</0> en het <1>Privacybeleid</1>."
+
+#: src/components/LabelingServiceCard/index.tsx:62
+msgid "By {0}"
+msgstr "Door {0}"
+
+#: src/view/com/composer/photos/OpenCameraBtn.tsx:72
+msgid "Camera"
+msgstr "Camera"
+
+#: src/view/com/modals/AddAppPasswords.tsx:180
+msgid "Can only contain letters, numbers, spaces, dashes, and underscores. Must be at least 4 characters long, but no more than 32 characters long."
+msgstr "Kan alleen letters, cijfers, spaties, streepjes en onderstrepingstekens bevatten. Moet minimaal 4 tekens lang zijn, maar niet meer dan 32 tekens."
+
+#: src/components/Menu/index.tsx:235
+#: src/components/Prompt.tsx:129
+#: src/components/Prompt.tsx:131
+#: src/components/TagMenu/index.tsx:267
+#: src/screens/Deactivated.tsx:161
+#: src/screens/Profile/Header/EditProfileDialog.tsx:220
+#: src/screens/Profile/Header/EditProfileDialog.tsx:228
+#: src/view/com/composer/Composer.tsx:684
+#: src/view/com/modals/ChangeEmail.tsx:213
+#: src/view/com/modals/ChangeEmail.tsx:215
+#: src/view/com/modals/ChangeHandle.tsx:141
+#: src/view/com/modals/ChangePassword.tsx:268
+#: src/view/com/modals/ChangePassword.tsx:271
+#: src/view/com/modals/CreateOrEditList.tsx:335
+#: src/view/com/modals/CropImage.web.tsx:97
+#: src/view/com/modals/InAppBrowserConsent.tsx:75
+#: src/view/com/modals/InAppBrowserConsent.tsx:77
+#: src/view/com/modals/LinkWarning.tsx:105
+#: src/view/com/modals/LinkWarning.tsx:107
+#: src/view/com/modals/VerifyEmail.tsx:255
+#: src/view/com/modals/VerifyEmail.tsx:261
+#: src/view/com/util/post-ctrls/RepostButton.tsx:166
+#: src/view/screens/Search/Search.tsx:910
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/CreateOrEditList.tsx:340
+#: src/view/com/modals/DeleteAccount.tsx:174
+#: src/view/com/modals/DeleteAccount.tsx:296
+msgctxt "action"
+msgid "Cancel"
+msgstr "Annuleer"
+
+#: src/view/com/modals/DeleteAccount.tsx:170
+#: src/view/com/modals/DeleteAccount.tsx:292
+msgid "Cancel account deletion"
+msgstr "Annuleeer accountverwijdering"
+
+#: src/view/com/modals/ChangeHandle.tsx:137
+msgid "Cancel change handle"
+msgstr "Annuleer wijzigen handle"
+
+#: src/view/com/modals/CropImage.web.tsx:94
+msgid "Cancel image crop"
+msgstr "Annuleer afbeelding bijsnijden"
```

Comment:
```suggestion
msgstr "Annuleer het bijsnijden van de afbeelding"
```
(this is an aria-label, so can be wordy)

Reply:
Changed to "Annuleer bijsnijden van afbeelding"